live-examples/js-examples/string/string-iterator.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">const str = 'The quick red fox jumped over the lazy dog's back.';
```

Comment:
The example fails with "Error: unexpected token: identifier" because you need to escape the apostrophe in "dog's".